payment/src/main/java/com/kyriba/conference/payment/api/PaymentController.java

```diff
@@ -0,0 +1,103 @@
+package com.kyriba.conference.payment.api;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.kyriba.conference.payment.domain.PaymentMethodType;
+import com.kyriba.conference.payment.domain.PaymentStatus;
+import com.kyriba.conference.payment.domain.dto.PaymentDto;
+import com.kyriba.conference.payment.domain.vo.Amount;
+import com.kyriba.conference.payment.domain.dto.TicketDto;
+import lombok.AllArgsConstructor;
+import lombok.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.time.Month;
+import java.util.Collections;
+import java.util.Currency;
+import java.util.List;
+import java.util.Locale;
+
+import static com.kyriba.conference.payment.domain.PaymentStatus.*;
+import static org.springframework.http.MediaType.APPLICATION_JSON_UTF8_VALUE;
+
+
+/**
+ * @author Igor Lizura
+ */
+@RestController
+@AllArgsConstructor
+@RequestMapping("/v1/payment")
+public class PaymentController {
+    @ResponseBody
+    @GetMapping(produces = APPLICATION_JSON_UTF8_VALUE)
+    public ResponseEntity<List<PaymentDto>> getPayments() {
```

Comment:
There is no need to wrap into response entity.

Reply:
done